agent/lib/kontena/launchers/etcd.rb

```diff
@@ -85,38 +85,47 @@ def create_data_container(image)
     # @param [String] image
     # @param [Hash] info
     def create_container(image, info)
```

Comment:
just thinking return values when container exists, should we always return container?